action.yml

```
@@ -13,6 +13,11 @@ inputs:
     default: false
     required: false
 
+outputs:
+  new-labels:
+    description: 'List of all new labels'
+  all-labels:
+    description: 'List of all labels'
```

I think the following wording is more informative
```suggestion
outputs:
  new-labels:
    description: 'A comma-separated list of all new labels'
  all-labels:
    description: 'A comma-separated list of all labels that the PR contains'
```